web3swift/src/ENS/EthereumNameService.swift

```diff
@@ -65,69 +58,57 @@ public class EthereumNameService: EthereumNameServiceProtocol {
         }
     }
 
-    required public init(
-        client: EthereumClientProtocol,
-        registryAddress: EthereumAddress? = nil,
-        maximumRedirections: Int = 5
-    ) {
+    required public init(client: EthereumClientProtocol,
+                         registryAddress: EthereumAddress? = nil,
+                         maximumRedirections: Int = 5) {
         self.client = client
         self.registryAddress = registryAddress
         self.maximumRedirections = maximumRedirections
     }
 
-    public func resolve(
-        address: EthereumAddress,
-        mode: ResolutionMode,
-        completion: @escaping ((EthereumNameServiceError?, String?) -> Void)
-    ) {
-        guard
-            let network = client.network,
-            let registryAddress = self.registryAddress ?? ENSContracts.registryAddress(for: network) else {
-                return completion(EthereumNameServiceError.noNetwork, nil)
-            }
+    public func resolve(address: EthereumAddress,
+                        mode: ResolutionMode,
+                        completionHandler: @escaping(Result<String, EthereumNameServiceError>) -> Void) {
+        guard let network = client.network,
+              let registryAddress = self.registryAddress ?? ENSContracts.registryAddress(for: network) else {
+            completionHandler(.failure(.noNetwork))
+            return
+        }
 
         Task {
             do {
-                let resolver = try await getResolver(
-                    for: address,
-                    registryAddress: registryAddress,
-                    mode: mode
-                )
+                let resolver = try await getResolver(for: address,
+                                                     registryAddress: registryAddress,
+                                                     mode: mode)
```

Comment:
are you running formatter on your side? Because it's polluting all meaningful changes for reviewing

Reply:
Yes its formatter, wasn't intended though.

Reply:
great 👍